test/testprocessexecutor.cpp

```diff
@@ -58,7 +58,7 @@ class TestProcessExecutor : public TestFixture {
      * Execute check using n jobs for y files which are have
      * identical data, given within data.
      */
-    void check(unsigned int jobs, int files, int result, const std::string &data, const CheckOptions &opt = {}) {
+    void check(unsigned int jobs, int files, int result, const std::string &data, CheckOptions opt = default_{}) {
```

Comment:
Yeah - the const ref was obviously not necessary but I think I started out without designated initialization. Maybe that might "fix" that compiler issues I was having (which is actually a bug in the compiler).

Reply:
Or I changed it because of the clang-tidy warning...my memory has been getting more spotty recently...